rakelib/output.rake

```diff
@@ -0,0 +1,94 @@
+if File.directory? 'Tests/Expected'
+  MODULE_INPUT_PATH = 'Fixtures/stub-env/Modules'
+  MODULE_OUTPUT_PATH = 'Fixtures/stub-env'
+  SDKS = {
+    :macosx => 'x86_64-apple-macosx10.12',
+    :iphoneos => 'armv7s-apple-ios10.0'
+  }
+  TOOLCHAINS = {
+    :swift2 => {
+      :module_path => "#{MODULE_OUTPUT_PATH}/swift2.3",
+      :toolchain => 'com.apple.dt.toolchain.Swift_2_3'
+    },
+    :swift3 => {
+      :module_path => "#{MODULE_OUTPUT_PATH}/swift3",
+      :toolchain => 'com.apple.dt.toolchain.XcodeDefault'
+    }
+  }
+
+  namespace :output do
+    desc 'Compile modules'
+    task :modules do |task|
+      Utils.print_info 'Compile output modules'
+
+      # macOS
+      modules = ['PrefsWindowController']
+      modules.each do |m|
+        puts "Compiling module #{m}… (macos)"
+        compile_module(m, :macosx, task)
+      end
+
+      # iOS
+      modules = ['CustomSegue', 'LocationPicker', 'SlackTextViewController']
+      modules.each do |m|
+        puts "Compiling module #{m}… (ios)"
+        compile_module(m, :iphoneos, task)
+      end
+
+      # delete swiftdoc
+      Dir.glob("#{MODULE_OUTPUT_PATH}/*.swiftdoc").each do |f|
+        FileUtils.rm_rf(f)
+      end
+    end
+
+    desc 'Compile output'
+    task :compile => :modules do |task|
+      Utils.print_info 'Compiling template output files'
+
+      exit Dir.glob('Tests/Expected/**/*.swift').map { |f|
```

Comment:
The argument can be a boolean or an integer. If it's boolean, then true means success. If it's an integer than 0 means success.